.github/workflows/eslint.yml

```diff
@@ -28,14 +28,9 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v3
       - name: Install Bun
-        run: npm install -g bun
-      - name: Install dependencies and @microsoft/eslint-formatter-sarif
-        run: bun install @microsoft/eslint-formatter-sarif
+        run: npm install -g pnpm
+      - name: Install dependencies
+        run: pnpm install
       - name: Run ESLint
-        run: SKIP_ENV_VALIDATION=true bun lint . --format @microsoft/eslint-formatter-sarif --output-file eslint-results.sarif
+        run: SKIP_ENV_VALIDATION=true pnpm lint
```

Comment:
why is the microsoft lint formatter gone?

Reply:
It was needed for sending the output somwhere, idk why I put it there.